src/rc-message/rc-message.ts

```diff
@@ -1,9 +1,9 @@
 import { XMLBuilder, XMLParser } from "fast-xml-parser";
 
-interface HDR {
+export interface HDR {
   [key: string]: string | undefined;
 }
-interface BDY {
+export interface BDY {
```

Comment:
I don't see you `import` this any where. 
or are you planning to use it in 3rd party apps (like Embeddable)? I am OK with that then.